src/v2/Apps/ViewingRoom/__tests__/ViewingRoomApp.jest.tsx

```diff
@@ -33,6 +34,56 @@ describe("ViewingRoomApp", () => {
     window.history.pushState({}, "Viewing Room Title", slug)
   })
 
+  // DRAFT viewing room
+  describe("with draft viewing room and user that has access to viewing rooms partner", () => {
+    beforeEach(() => {
+      jest.mock("v2/Utils/user", () => ({
```

Comment:
I was hoping that this way I can mock `userHasAccessToPartner` that gets called in the app from the ViewingRoomApp but that did not work. I'll need to research more or maybe pair with somebody on that tomorrow